library/src/main/java/net/yslibrary/licenseadapter/BaseLibrary.java

```diff
@@ -52,14 +156,13 @@ public boolean equals(Object o) {
 
     BaseLibrary that = (BaseLibrary) o;
 
-    return name.equals(that.name) && author.equals(that.author) && license.equals(that.license);
+    return name.equals(that.name) && author.equals(that.author);
```

Comment:
This was a mistake. The `Library` is still the same library whether or not the license is different. Removing license checks enables caching in a map before and after the license has changed.